### DISCUSSION TITLE
新版本开发中

### DISCUSSION THREAD

**Discussion by andywang425** [andywang425/BLTH#395] [General]
目前我正在用全新的技术栈重写整个脚本，预计在八月末之前发布。当前脚本的大部分功能都会被移植过去。

因此现在不会花太多精力在旧版本的维护上，可能无法及时修复Bug。如果有想要帮忙的欢迎PR。

**Comment by Themoonlove** [comment]
新版界面关不掉咋办

**Comment by yangyangbuxiangyang** [comment]
不小心更新了，我现在就是后悔，非常非常的后悔。

**Comment by Victor-Huo** [comment]


**Reply by andywang425** [reply]
会做这个功能，但不确定什么时候能做好。我马上就要开学了，到时候不会有太多时间来维护这个项目。

还有别在这里提功能请求，开个新的discussion或者issue。

**Reply by Victor-Huo** [reply]
好的不好意思
